.github/workflows/perftest.yml

```diff
@@ -74,15 +74,15 @@ jobs:
         pushd .
         cd ./fio
         ./configure
-        make
+        make -j$(nproc)
         sudo make install
         popd
     - name: (rawstor_ost) start rawstor_ost
       if: "${{ matrix.disable-ost == '' }}"
       run: |
-        mkdir /tmp/objects
-        truncate -s 2G /tmp/objects/01988675-e1e5-7788-9ed4-2f2d30910d23
-        ost 8080 /tmp/objects/ &
+        mkdir ${GITHUB_WORKSPACE}/objects
```

Comment:
Let's create explicit tmpfs mountpoint? It'd be far faster and cleaner to use. We have plenty of RAM on this runner.